x11-terms/mrxvt/mrxvt-0.5.4.ebuild

```diff
@@ -43,11 +42,6 @@ PATCHES=(
 src_prepare() {
 	default
 	eautoreconf
-
-	if use elibc_uclibc && use truetype; then
-		# It is stated in the README "Multichar support under XFT requires GNU iconv"
```

Comment:
seems odd this actually needs to go. you can probably get away with a cheesy has_version here libiconv as glibc will always (probably!) provide it and people don't change libcs.